CHANGELOG.md

```diff
@@ -21,6 +21,10 @@
 
 ## v3.21.2
```

Comment:
```suggestion
## v3.21.3
```

Reply:
🤦 wow, clearly I should stop for the night.  thanks